README.md

```diff
@@ -184,6 +184,24 @@ dangerousDisableContainerSupport: true
 Alternatively, the presence of ``-XX:MaxRAM=`` prefix in either static or custom jvm opts will also disable this
 behavior.
 
+### Graal Native Image support
+
+Go-java-launcher supports running a native image of a Java application. For example:
+
+```yaml
+experimental:
+  executionMode: native
+  nativeImageExecutablePath: service/bin/<native-image-executable>
```

Comment:
A bit separate, but is there a way we could avoid having devs/operators figure out the correct path/guess the executable name here? (either having a reasonable default, or something else, e.g. a metadata file that contains the name of the executable, embedded within the service, or just renaming the file to something standard in our build tooling?)